frontend/src/metabase/static-viz/components/XYChart/XYChart.tsx

```diff
@@ -202,6 +203,16 @@ export const XYChart = ({
       />
 
       <Group left={valuesLeftOffset}>
+        <Group top={margin.top} left={xMin}>
```

Comment:
Fix the axis covered by grid lines.

#### After
![image](https://user-images.githubusercontent.com/1937582/188827468-7675183f-4efa-44f1-bdf7-75b2d151ace2.png)

#### Before
![image](https://user-images.githubusercontent.com/1937582/188827248-7c9ec3ae-2002-404b-a645-7adedb2b7fe2.png)